ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -141,7 +141,9 @@ export default function BuildQuote({
   const toTokenIsNotDefault =
     selectedToToken?.address &&
     !isSwapsDefaultTokenAddress(selectedToToken?.address, chainId);
-  const occurances = Number(selectedToToken?.occurances || 0);
+  const occurrences = Number(
+    selectedToToken?.occurances || selectedToToken?.occurrences || 0,
```

Comment:
Is this change because the `occurances` property will be spelled differently on api v2?

Reply:
Yup, v2 uses correct naming, but here I support both v1 and v2 versions.